rskj-core/src/main/java/co/rsk/RskContext.java

```diff
@@ -1886,11 +1887,14 @@ protected SyncProcessor getSyncProcessor() {
 
     private PeersInformation getPeersInformation() {
         if (peersInformation == null) {
+            SyncConfiguration syncConfiguration = getSyncConfiguration();
+
             peersInformation = new PeersInformation(
                     getChannelManager(),
-                    getSyncConfiguration(),
+                    syncConfiguration,
                     getBlockchain(),
-                    getPeerScoringManager());
+                    getPeerScoringManager(),
+                    syncConfiguration.getTopBest());
```

Comment:
we alrady pass `syncConfiguration` as a param to the `PeersInformation` ctor. This extra param might not be needed

Reply:
Addressed